### PR TITLE
added SVG for Groq model model providers

### PR DESCRIPTION
**Pull request by paresh2806**
#1432  #1447 
This PR adds support for the GROQ LLM (Large Language Model).

Groq is an AI solutions company delivering ultra-low latency inference with the first-ever LPU™ Inference Engine. The Groq API enables developers to integrate state-of-the-art LLMs, such as Llama-2 and llama3-70b-8192, into low latency applications with the request limits specified below. Learn more at [groq.com](https://groq.com/).
Supported Models

| ID                   | Requests per Minute | Requests per Day | Tokens per Minute |
|----------------------|---------------------|------------------|-------------------|
| gemma-7b-it          | 30                  | 14,400           | 15,000            |
| gemma2-9b-it         | 30                  | 14,400           | 15,000            |
| llama3-70b-8192      | 30                  | 14,400           | 6,000             |
| llama3-8b-8192       | 30                  | 14,400           | 30,000            |
| mixtral-8x7b-32768   | 30                  | 14,400           | 5,000             |

### CODE REVIEW

File: rag/llm/__init__.py [hunk 1]
```diff
@@ -32,7 +32,8 @@
     "Jina": JinaEmbed,
     "BAAI": DefaultEmbedding,
     "Mistral": MistralEmbed,
-    "Bedrock": BedrockEmbed
+    "Bedrock": BedrockEmbed,
+    "Groq": Groqchat
```
Comment (KevinHuSh) [inline rag/llm/__init__.py]:
```suggestion
    "Groq": GroqChat
```

File: rag/llm/chat_model.py [hunk 2]
```diff
@@ -681,4 +682,63 @@ def chat_streamly(self, system, history, gen_conf):
         except Exception as e:
             yield ans + "\n**ERROR**: " + str(e)
 
-        yield  response._chunks[-1].usage_metadata.total_token_count
\ No newline at end of file
+        yield  response._chunks[-1].usage_metadata.total_token_count
+
+
+
+class Groqchat:
```
Comment (KevinHuSh) [inline rag/llm/chat_model.py]:
```suggestion
class GroqChat:
```

File: api/db/init_data.py [hunk 3]
```diff
@@ -933,6 +939,42 @@ def init_llm_factory():
             "tags": "TEXT EMBEDDING",
             "max_tokens": 2048,
             "model_type": LLMType.EMBEDDING.value
+        },
+        # ------------------------ Groq -----------------------
+        {
+            "fid": factory_infos[18]["name"],
+            "llm_name": "gemma-7b-it",
+            "tags": "LLM,CHAT,15k",
+            "max_tokens": 15000,
```
Comment (KevinHuSh) [inline api/db/init_data.py]:
```suggestion
            "max_tokens": 8192,

```

File: api/db/init_data.py [hunk 4]
```diff
@@ -933,6 +939,42 @@ def init_llm_factory():
...
+            "llm_name": "gemma-7b-it",
+            "tags": "LLM,CHAT,15k",
+            "max_tokens": 15000,
+            "model_type": LLMType.CHAT.value
+        },
+        {
+            "fid": factory_infos[18]["name"],
+            "llm_name": "gemma2-9b-it",
+            "tags": "LLM,CHAT,15k",
+            "max_tokens": 15000,
```
Comment (KevinHuSh) [inline api/db/init_data.py]:
```suggestion
            "max_tokens": 8192,

```

File: api/db/init_data.py [hunk 5]
```diff
@@ -933,6 +939,42 @@ def init_llm_factory():
...
+            "model_type": LLMType.CHAT.value
+        },
+        {
+            "fid": factory_infos[18]["name"],
+            "llm_name": "llama3-70b-8192",
+            "tags": "LLM,CHAT,6k",
+            "max_tokens": 6000,
```
Comment (KevinHuSh) [inline api/db/init_data.py]:
```suggestion
            "max_tokens": 8192,

```

File: api/db/init_data.py [hunk 6]
```diff
@@ -933,6 +939,42 @@ def init_llm_factory():
...
+            "model_type": LLMType.CHAT.value
+        },
+        {
+            "fid": factory_infos[18]["name"],
+            "llm_name": "llama3-8b-8192",
+            "tags": "LLM,CHAT,30k",
+            "max_tokens": 30000,
```
Comment (KevinHuSh) [inline api/db/init_data.py]:
```suggestion
            "max_tokens": 8192,

```

File: api/db/init_data.py [hunk 7]
```diff
@@ -933,6 +939,42 @@ def init_llm_factory():
...
+            "model_type": LLMType.CHAT.value
+        },
+        {
+            "fid": factory_infos[18]["name"],
+            "llm_name": "mixtral-8x7b-32768",
+            "tags": "LLM,CHAT,5k",
+            "max_tokens": 5000,
```
Comment (KevinHuSh) [inline api/db/init_data.py]:
```suggestion
            "max_tokens": 32768,

```

**Comment by lionkingc** [comment]
please add the newest model : 
Model ID: llama-3.1-70b-versatile
Model ID: llama-3.1-8b-instant

**Pull request merged**